parser/parser.go

```diff
@@ -1191,3 +1191,12 @@ func Parse(t Tokens) (ast.Node, error) {
 
 	return expr, nil
 }
+
```

Comment:
I guess `parser` should also be internal, but I want to move it later in a separate change that cleans up all remaining unnecessarily exported stuff.